router-beta1-test/README.md

```diff
@@ -1,4 +1,4 @@
-# Deis Router 2.0.0-alpha
+# Deis Router 2.0.0-beta
 
 WARNING: this chart is for testing only! Features may not work and there are likely to be bugs.
```

Comment:
Again... I just followed the pattern from the `workflow-beta1-test` chart with the assumption that it was correct.